src/sunburst.js

```diff
@@ -22,6 +22,7 @@ export default Kapsule({
     sort: { onChange(_, state) { state.needsReparse = true }},
     label: { default: d => d.name },
     labelOrientation: { default: 'auto' }, // angular, radial, auto
+    nodeCss: { default: d => '' }, // Additional css classes to add on each slice node
```

Comment:
Please update the type definitions (on `index.d.ts`) to include this new method.

Reply:
Updated.